doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,113 @@
+factors](#compute-cost-factors) for more information on compute costs for replicas.
+
+### Compute cost factors
+
+{{< note >}}
+  Materialize provides up-to-date results while also providing
+  strong [consistency guarantees](/get-started/isolation-level/), including
+  strict serializability (the default level). Materialize maintains these
+  up-to-date results via incrementally maintained
+  [indexes](/concepts/indexes/) and incrementally maintained [materialized
+  views](/concepts/views) that perform computation upon insert/update/delete. That is:
+
+  - Materialize performs work on "writes"; i.e., Materialize updates the results
+    when data is ingested.
+
+  - Materialize performs **no** work on "reads" from these objects; i.e.,
+    [`SELECT`s](/sql/select/) from these objects, including ad-hoc queries, are
+    free.
+
+  - Materialize is always "on".
+
+{{</ note >}}
+
+Factors that contribute to compute usage include:
+
+| Factor | Notes       |
+|-------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Each [replica for a cluster](/sql/create-cluster/#replication-factor). | The credit usage for a cluster is measured at a one second granularity for each cluster replica. For a cluster replica, its credit usage begins when it is provisioned and ends when it is deprovisioned. |
+| [Indexes](/concepts/indexes/) and [materialized views](/concepts/views) | As data changes (insert/update/delete), [indexes](/concepts/indexes/) and [materialized views](/concepts/views) perform incremental updates to provide up-to-date results. That is: <br>• Indexes and materialized views perform work on "writes"<br>• Indexes and materialized views perform **no** work on "reads" from these objects; i.e., [`SELECT`s](/sql/select/) and [`SUBSCRIBE`s](/sql/subscribe/) from these objects are **free**. |
+| [Sources](/concepts/sources/) | Depends. |
+| [`SELECT`s](/sql/select/) and [`SUBSCRIBE`s](/sql/subscribe/)  |• [`SELECT`s](/sql/select/) and [`SUBSCRIBE`s](/sql/subscribe/) that do not use indexes and materialized views perform work. <br>• [`SELECT`s](/sql/select/) and [`SUBSCRIBE`s](/sql/subscribe/) that use indexes and materialized views are **free**.|
+| [Sinks](/concepts/sinks/) | Only small CPU/memory costs.|
+
+### Compute cost reductions
```

Comment:
Does this belong in a page about billing? I'd expect something more in line with the current pricing page (_e.g._ with the available cluster sizes).

Reply:
This can definitely go away now. This was before this page had a definitive shape.

Reply:
As for cluster size, yup -- I saw https://materialize.com/docs/sql/create-cluster/#size -- but, it's kind of unwieldy looking list.  I figured I'll redo that list in a separate pull and share it across various pages, including this.